src/ripple/app/misc/impl/ValidatorSite.cpp

```diff
@@ -210,17 +214,30 @@ ValidatorSite::makeRequest (
     fetching_ = true;
     sites_[siteIdx].activeResource = resource;
     std::shared_ptr<detail::Work> sp;
+    auto timeoutCancel =
+        [this] ()
+        {
+            std::unique_lock <std::mutex> lock_state{state_mutex_};
+            error_code ec;
+            timer_.cancel_one(ec);
+        };
```

Comment:
This overload of `cancel_one` is deprecated. You can use the non-error_code overload. eg.  `timer_.cancel_one();`

Reply:
fixed

Reply:
This doesn't appear to be fixed yet.